docs/conf.py

```diff
@@ -356,4 +356,13 @@
     ("py:mod", "aiohttp"),  # undocumented, no `.. currentmodule:: aiohttp` in docs
     ("py:class", "aiohttp.SimpleCookie"),  # undocumented
     ("py:class", "aiohttp.web.RequestHandler"),  # undocumented
+    ("py:meth", "aiohttp.ClientSession.request"),  # undocumented
+    ("py:class", "aiohttp.protocol.HttpVersion"),  # undocumented
+    ("py:class", "aiohttp.ClientRequest"),  # undocumented
+    ("py:class", "Payload"),  # undocumented
+    ("py:class", "aiohttp.abc.AbstractResolver"),  # undocumented
+    ("py:func", "aiohttp.ws_connect"),  # undocumented
+    ("py:meth", "start"),  # undocumented
+    ("py:exc", "aiohttp.ServerDisconnectionError"),  # undocumented
+    ("py:exc", "ClientHttpProxyError"),  # undocumented
```

Comment:
Same: it's not clear where it's defined.

Reply:
`aiohttp. ClientHttpProxyError`?

Reply:
Maybe, check via REPL